### PR TITLE
[incubator/druid] Specify these configurations about druid_indexer_logs_xxx

### PR DESCRIPTION
**Pull request by asdf2014**
No

#### What this PR does / why we need it:

Specify these configurations about druid_indexer_logs_xxx

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Special notes for your reviewer:

@maver1ck @AWaterColorPen

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: incubator/druid/values.yaml [hunk 1]
```diff
@@ -23,6 +23,8 @@ configVars:
   druid_indexer_runner_javaOptsArray: '["-server", "-Xmx1g", "-Xms1g", "-XX:MaxDirectMemorySize=3g", "-Duser.timezone=UTC", "-Dfile.encoding=UTF-8", "-Djava.util.logging.manager=org.apache.logging.log4j.jul.LogManager"]'
   druid_indexer_fork_property_druid_processing_buffer_sizeBytes: '268435456'
   druid_storage_type: local
+  druid_indexer_logs_type: file
+  druid_indexer_logs_directory: /opt/data/indexing-logs
```
Comment (AWaterColorPen) [inline incubator/druid/values.yaml:27]: There is no `/opt/data/indexing-logs` directory in the docker image. 
Will the druid create the `/opt/data/indexing-logs` directory if it is non-existent?

Comment (asdf2014) [reply incubator/druid/values.yaml:27]: @AWaterColorPen Yes, It will create the directory automatically.

**Comment by asdf2014** [comment]
Hi, @AWaterColorPen . Do you have any other comments?

File: incubator/druid/values.yaml (hunk 1 above)
Comment (AWaterColorPen) [reply incubator/druid/values.yaml:27]: great!

**Comment by AWaterColorPen** [comment]
@asdf2014 . It is good to merge

**Pull request merged**